paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -535,60 +392,6 @@ internal class PaymentOptionsAdapter(
         }
     }
 
-    internal enum class ViewType {
-        SavedPaymentMethod,
-        AddCard,
-        GooglePay,
-        Link
-    }
-
-    internal sealed class Item {
-        abstract val viewType: ViewType
```

Comment:
Moved to its own file and renamed to `PaymentOptionsItem` (because `PaymentOption` already exists — other name ideas welcome!)